check/src/validators_mev.rs

```diff
@@ -0,0 +1,49 @@
+use log::info;
+use rust_decimal::prelude::*;
+use solana_client::rpc_client::RpcClient;
+use tokio_postgres::Client;
+
+pub async fn check_mev(psql_client: &Client, rpc_client: &RpcClient) -> anyhow::Result<()> {
+    info!("Checking `mev` data table about epoch in DB");
+
+    let rows = psql_client
+        .query(
+            "SELECT epoch, MAX(epoch_slot) as epoch_slot
+                    FROM mev
+                    WHERE epoch = (SELECT MAX(epoch) FROM mev)
+                    GROUP BY epoch;",
+            &[],
+        )
+        .await?;
+
+    assert!(rows.len() <= 1);
+
+    match rows.iter().next() {
+        Some(row) => {
+            let sql_epoch: Decimal = row.get("epoch");
+            let sql_slot: Decimal = row.get("epoch_slot");
+
+            let epoch_data = rpc_client.get_epoch_info()?;
+            let current_epoch = Decimal::from(epoch_data.epoch);
+            let current_slot = Decimal::from(epoch_data.absolute_slot);
+
+            info!(
+                "DB stores last MEV epoch: {sql_epoch}, slot: {sql_slot}, on-chain epoch: {current_epoch}, slot: {current_slot}"
+            );
+
+            // If the current epoch is bigger than the lastly stored MEV epoch, we need to proceed
+            if current_epoch > sql_epoch {
```

Comment:
isn't this off by 1?

Reply:
(also, linguistic nitpick: `greater`)

Reply:
Yes, you're right. Thanks. I haven't realized that while not having access to prod data. I updated the code.